public/locales/en-GB/app.json

```diff
@@ -42,6 +42,7 @@
   "Display name": "Display name",
   "Download debug logs": "Download debug logs",
   "Element Call Home": "Element Call Home",
+  "Enable keyboard shortcuts": "Enable keyboard shortcuts",
```

Comment:
"Single-key keyboard shortcuts" perhaps? (since it's specifically ones that activate without modifier keys held)

Reply:
I can't speak for product but the single-key shortcuts are the ones that cause issues for a11y (especially spacebar since that's used for interacting with elements on the page on place of a click).

Reply:
Yeah, agreed - we can make this exclusively about single-key shortcuts.

Reply:
Ok cool, will change the wording then.

Reply:
Done